packages/design-system/src/components/Promo/style.scss

```diff
@@ -0,0 +1,156 @@
+@import '../../scss/variables/colors';
+
+$base-class: 'promo';
+
+.#{$base-class} {
+  position: relative;
+  padding: 20px;
+  overflow: hidden;
```

Comment:
Seems like it was copied from Banner component style. `overflow` property is not inherited by children

Reply:
removed